src/components/Disclaimer/Consent.tsx

```diff
@@ -0,0 +1,123 @@
+import { Dialog, Transition } from "@headlessui/react"
+import { FC, Fragment, useEffect, useState } from "react"
+
+import useStorage from "@/hooks/util/useStorage"
+
+import Button from "@/components/Button"
+
+export const Consent: FC = () => {
+  const { getItem, setItem } = useStorage()
+  const expectedValue = "agreed"
+  const agreed = getItem("disclaimer", "local")
+  const [isOpen, setIsOpen] = useState(false)
+
+  function closeModal() {
+    setItem("disclaimer", expectedValue, "local")
+    setIsOpen(false)
+  }
+
+  useEffect(() => {
+    if (!(agreed !== undefined && agreed === expectedValue)) {
+      setIsOpen(true)
+    }
+  }, [setIsOpen, agreed, expectedValue])
+
+  return (
+    <>
+      <Transition appear show={isOpen} as={Fragment}>
+        <Dialog
+          as="div"
+          className="pointer-events: none relative z-10"
+          onClose={() => null}
+        >
+          <Transition.Child
+            as={Fragment}
+            enter="ease-out duration-300"
+            enterFrom="opacity-0"
+            enterTo="opacity-100"
+            leave="ease-in duration-200"
+            leaveFrom="opacity-100"
+            leaveTo="opacity-0"
+          >
+            <div className="bg-purple-gray-900/80 fixed inset-0 opacity-100 backdrop-blur-sm" />
+          </Transition.Child>
+
+          <div className="fixed inset-0 overflow-y-auto">
+            <div className="flex min-h-full items-center justify-center p-4 text-center">
+              <Transition.Child
+                as={Fragment}
+                enter="ease-out duration-300"
+                enterFrom="opacity-0 scale-95"
+                enterTo="opacity-100 scale-100"
+                leave="ease-in duration-200"
+                leaveFrom="opacity-100 scale-100"
+                leaveTo="opacity-0 scale-95"
+              >
+                <Dialog.Panel className="min-w-[360px] !max-w-prose transform overflow-hidden rounded-md bg-pink-900 p-8 text-justify align-middle transition-all">
+                  <Dialog.Title
+                    as="h3"
+                    className="text-2xl font-medium leading-6 text-white"
+                  >
+                    DISCLAIMER
+                  </Dialog.Title>
+                  <div className="mt-6">
+                    <p className="mb-5 text-sm leading-6 text-slate-300">
```

Comment:
One of the things that I love most about Tailwind is the way that color schemes work in the framework. 

I find that if you have a container with a background like `pink-900`, your text will look awesome in that same color but in a lighter shade. The opposite will generally hold true on a light background (e.g. `bg-pink-100 text-pink-800`). 

The color schemes that come with tailwind are awesome, but the colors are meant to work well in combination with other built-in colors. This slate color looks a bit off because it isn't part of our custom scheme. 

Try doing `text-pink-100`, I think it looks great here.